modules/mtbseq/tbgroups/tbgroups.nf

```diff
@@ -35,8 +35,8 @@ process TBGROUPS {
     sleep \$[ ( \$RANDOM % 10 )  + 1 ]s
 
     mkdir Groups
-    touch Groups/${params.mtbseq_project_name}_joint_[mincovf]_[mincovr]_[minfreq]_[minphred20]_samples_amended_[unambig]_phylo_[window].matrix
-    touch Groups/${params.mtbseq_project_name}_joint_[mincovf]_[mincovr]_[minfreq]_[minphred20]_samples_amended_[unambig]_phylo_[window]_[distance].groups
+    touch Groups/${params.mtbseq_project_name}_joint_cf4_cr4_fr75_ph4_samples6_amended_u95_phylo_w12.matrix
```

Comment:
Perhaps we should use `samples5` as defined in the `mock_data` folder 

https://github.com/emilyncosta/mtbseq-nf/tree/dev/data/mock_data

Reply:
Therefore `sample5` would be used throughout the stubs in other processes as well.